ingredient/ingredient-service-bus-namespace/package.json

```diff
@@ -35,7 +35,8 @@
   },
   "dependencies": {
     "@azbake/arm-helper": "^0.1.55",
-    "@azure/arm-servicebus": "^3.2.0"
+    "@azure/arm-servicebus": "^3.2.0",
+    "@azbake/ingredient-event-hub-namespace": "^0.1.55"
```

Comment:
shouldn't include one ingredient as a dep for another. Use the system for getting a util function handler, with instructions that this ingredient requires including the other ingredient.

Reply:
See other comment